jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -331,20 +332,23 @@ private void pushBaseImageLayers() {
     ProgressEventDispatcher.Factory childProgressDispatcherFactory =
         Verify.verifyNotNull(rootProgressDispatcher).newChildProducer();
 
-    results.baseImageLayerPushResults =
+    results.baseImagesAndbaseImageLayerPushResults =
```

Comment:
```suggestion
    results.baseImagesAndBaseImageLayerPushResults =
```
but because this time it's all about base images, I think we can go with
```suggestion
    results.baseImagesAndLayerPushResults =
```